stackslib/src/chainstate/coordinator/mod.rs

```diff
@@ -3274,7 +3274,7 @@ impl<
                     if let Some(ref mut estimator) = self.cost_estimator {
                         let stacks_epoch = self
                             .sortition_db
-                            .index_conn()
+                            .index_handle_at_tip()
```

Comment:
Can you avoid using this in consensus code?  It can panic instead of propagating an underlying DB error.  Perhaps add a fallible variant of `index_handle_at_tip`?